stripe/src/main/java/com/stripe/android/cards/DefaultCardAccountRangeStore.kt

```diff
@@ -0,0 +1,44 @@
+package com.stripe.android.cards
+
+import android.content.Context
+import com.stripe.android.model.CardMetadata
+import com.stripe.android.model.parsers.AccountRangeJsonParser
+import org.json.JSONObject
+
+internal class DefaultCardAccountRangeStore(
+    private val context: Context
+) : CardAccountRangeStore {
+    private val accountRangeJsonParser = AccountRangeJsonParser()
+
+    private val prefs by lazy {
+        context.getSharedPreferences(PREF_FILE, Context.MODE_PRIVATE)
+    }
+
+    override suspend fun get(bin: String): List<CardMetadata.AccountRange> {
```

Comment:
Do we need to handle concurrency here? possibly overkill? I can imagine a scenario where we ned up firing multiple of the same request, but I'm not sure exactly where we want to handle that.

Reply:
We might need to, but I'll come back to that in a later PR